admin/release-packaging/README.rst

```diff
@@ -0,0 +1,3 @@
+Packages for Flocker that aren't available in each supported Operating System.
```

Comment:
This should probably be something about packages that point at our repositories, instead.